src/map/pc.c

```diff
@@ -4847,10 +4847,40 @@ int pc_isUseitem(struct map_session_data *sd,int n)
 			if( pc_issit(sd) )
 				return 0;
 			break;
+
+		case ITEMID_GIANT_FLY_WING: {
+			struct party_data *p;
+
+			if (!sd->status.party_id) {
+				clif->msgtable(sd, MSG_PARTY_MEMBER_NOT_SUMMONED);
+				break;
```

Comment:
Just to make sure, is this intended? Should the item still be used even if this message is displayed, or should it `return 0` here?